concent_api/test_server_script.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+from json import JSONDecodeError
+import datetime
+import random
+
+import requests
+
+
+DEFAULT_HEADERS = {
+    'Content-Type': 'application/json',
+    'concent-client-public-key': '85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2UssnPrCjHKTi5susO3wrr/q07eswumbL82b4HgOw==',
+}
+
+
+class InvalidOption(Exception):
+    pass
+
+
+def create_data(message_type):
+    current_time = int(datetime.datetime.now().timestamp())
+    data = {
+        "type":         message_type,
+        "timestamp":    current_time,
+        "message_task_to_compute": {
+            "type":         "MessageTaskToCompute",
+            "timestamp":    current_time,
+            "task_id":      5,
+            "deadline":     current_time + 6000
+        }
+    }
+    if message_type == "MessageRejectReportComputedTask":
+        data = {
+            "type":         "MessageRejectReportComputedTask",
+            "timestamp":    current_time,
+            "reason":       "cannot-compute-task",
+            "message_cannot_commpute_task": {
+                "type":         "MessageCannotComputeTask",
+                "timestamp":    current_time,
+                "reason":       "provider-quit",
+                "task_id":      5
+            }
+        }
+    return data
+
+
+def api_request(host, endpoint, task_id = None, data = None, headers = None):
+    if headers is None:
+        headers = DEFAULT_HEADERS
+
+    if data is None:
+        print('Send message {}:'.format(endpoint))
+    else:
+        print('Send message {}:'.format(data['type']))
+    print("{}/api/v1/{}/".format(host, endpoint))
+
+    if data is None:
+        response = requests.post("{}/api/v1/{}/".format(host, endpoint), headers=headers)
+    else:
+        if 'message_task_to_compute' in data:
```

Comment:
This is a weird place to put it. Add `task_id` parameter to `create_data` and make it generate a message already with IDs. The function that sends a message should just accept it as is and not make assumptions about what's inside it.